### PR TITLE
SLVSCODE-699 limit binding suggestion notifications to 1

### PR DESCRIPTION
**Pull request by sophio-japharidze-sonarsource**

### CODE REVIEW

File: src/connected/autobinding.ts [hunk 1]
```diff
@@ -237,10 +234,11 @@ export class AutoBindingService {
         await this.bindingService.saveBinding(
           bindingSuggestion.sonarProjectKey, bindingSuggestion.connectionId, unboundFolder);
         break;
-      case CHOOSE_MANUALLY_ACTION:
+      case CHOOSE_MANUALLY_ACTION: {
```
Comment (thahnen) [inline src/connected/autobinding.ts:237]: Why are the curly braces added? I'm not familiar with TypeScript but I would guess that it is better for cases where multiple statements will follow before the break.
This is also the same question for the braces added below.

On the other hand, I could also guess it is to have the variable `targetConnection` in the local scope only to be discarded outside of the case statement.

**Review by thahnen — APPROVED** [review]
LGTM! Know the issue from Eclipse where we display way too many binding notifications as well 😶‍🌫️

File: src/connected/autobinding.ts (hunk 1 above)
Comment (sophio-japharidze-sonarsource) [reply src/connected/autobinding.ts:237]: https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.vscode%3Asonarlint-vscode&fixedInPullRequest=538

**Review by jblievremont — APPROVED** [review]
LGTM 🚀

**Comment by jblievremont** [comment]
> [3 Fixed issues](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.vscode%3Asonarlint-vscode&fixedInPullRequest=538)

💯 👍🏻 🤩

**Pull request merged**